muspinsim/mpi.py

```diff
@@ -98,10 +96,10 @@ def broadcast_terms(self, lst):
 
             try:
                 val = self.comm.bcast(val, root=0)
-            except OverflowError as e:
+            except OverflowError as exc:
                 raise OverflowError(
-                    "Overflow error broadcasting term {0}: {1}".format(val, str(e))
-                )
+                    f"Overflow error broadcasting term {val}: {str(exc)}"
```

Comment:
```suggestion
                    f"Overflow error broadcasting term {val}"
```
See other comment for discussion.